checkpoint_ngfw/help.md

```diff
@@ -833,8 +875,29 @@ to True in each action. That will effectively remove all other pending changes w
 plugin tries to publish its changes. A best practice is to have separate administrator accounts so that you can 
 better track changes done via the plugin or manually via SmartConsole.
 
+### Common Errors
+
+#### 403 forbidden
+
+If you are presented with a 403 forbidden error when running the connection test, the API hasn't been enabled.
+ 
+For more information on enabling the API visit here: https://community.checkpoint.com/t5/API-CLI-Discussion-and-Samples/Enabling-web-api/td-p/32641
```

Comment:
```suggestion
For more information on enabling the API visit [https://community.checkpoint.com/t5/API-CLI-Discussion-and-Samples/Enabling-web-api/td-p/32641]( https://community.checkpoint.com/t5/API-CLI-Discussion-and-Samples/Enabling-web-api/td-p/32641).
```

Reply:
Changed this a little to avoid the . at the end of the line. That way you can copy paste the URL easier if needed.